source/crud/crud.rst

```diff
@@ -227,6 +227,13 @@ Read
      */
     useCursor: Optional<Boolean>;
 
+    /**
+     * Enables users to specify an arbitrary string to help trace the operation through
+     * the database profiler, currentOp and logs. The default is not send a value.
+     *
+     * @see http://docs.mongodb.org/manual/reference/command/aggregate/
```

Comment:
Canonical URL is "docs.mongodb.com" and we should probably specify https everywhere. Let's change to: `https://docs.mongodb.com/manual/reference/command/aggregate/`.